src/docs/fr/profile/errors.md

```diff
@@ -0,0 +1,30 @@
+---
+path: "/profile/errors"
+subnav: "1/Profile/Profil/8"
+lang: "fr"
+title: "Errors (Translate)"
+---
+
+<helmet>
+<title> Profil - Errors (Translate)</title>
+</helmet>
+
+# Error Codes
+TEXT
```

Comment:
The French version of this page doesn't format into a table. My guess would be that the TEXT on line 13 is causing it.